[2024-06-24 14:31:22,420] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task 2023-06-17T00:00:00+00:00 [queued]>
[2024-06-24 14:31:22,478] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task 2023-06-17T00:00:00+00:00 [queued]>
[2024-06-24 14:31:22,493] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-06-24 14:31:22,494] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2024-06-24 14:31:22,494] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-06-24 14:31:22,551] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2023-06-17T00:00:00+00:00
[2024-06-24 14:31:22,577] {standard_task_runner.py:52} INFO - Started process 20756 to run task
[2024-06-24 14:31:22,594] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v5', 'second_task', '2023-06-17T00:00:00+00:00', '--job-id', '8341', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpw530yuz5', '--error-file', '/tmp/tmplwc18sll']
[2024-06-24 14:31:22,608] {standard_task_runner.py:77} INFO - Job 8341: Subtask second_task
[2024-06-24 14:31:23,002] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v5.second_task 2023-06-17T00:00:00+00:00 [running]> on host 799b1e36e3f1
[2024-06-24 14:31:23,270] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2023-06-17T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-06-17T00:00:00+00:00
[2024-06-24 14:31:23,272] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2024-06-24 14:31:23,278] {bash.py:158} INFO - Running command: echo hey, I am task2 and will be running after task1!
[2024-06-24 14:31:23,331] {bash.py:169} INFO - Output:
[2024-06-24 14:31:23,337] {bash.py:173} INFO - hey, I am task2 and will be running after task1!
[2024-06-24 14:31:23,340] {bash.py:177} INFO - Command exited with return code 0
[2024-06-24 14:31:23,454] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20230617T000000, start_date=20240624T143122, end_date=20240624T143123
[2024-06-24 14:31:23,587] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-24 14:31:23,628] {local_task_job.py:146} INFO - Task exited with return code 0
